sqlmesh/core/loader.py

```diff
@@ -73,17 +73,90 @@ def get_or_load_models(
         self, target_path: Path, loader: t.Callable[[], t.List[Model]]
     ) -> t.List[Model]:
         """Get or load all models from cache."""
+        pass
+
+    @abc.abstractmethod
+    def put(self, models: t.List[Model], path: Path) -> bool:
```

Comment:
Can we please add some docstrings. Eg. I'm not sure what does the boolean return value suppose to represent.